atc/db/build.go

```diff
@@ -172,15 +178,9 @@ func (b *build) ReapTime() time.Time          { return b.reapTime }
 func (b *build) Status() BuildStatus          { return b.status }
 func (b *build) IsScheduled() bool            { return b.scheduled }
 func (b *build) IsDrained() bool              { return b.drained }
-
-func (b *build) IsRunning() bool {
-	switch b.status {
-	case BuildStatusPending, BuildStatusStarted:
-		return true
-	default:
-		return false
-	}
-}
+func (b *build) IsRunning() bool              { return !b.completed }
```

Comment:
This change means we consider builds on which we've called `MarkAsAborted()` on as "Running", which is different from the existing behaviour.

Reply:
An easy fix would be to set the completed bool in `MarkAsAborted()`, assuming its safe to consider an abort as a possible path for a build to be completed